scripts/plugin-config.yml

```diff
@@ -15,4 +15,6 @@ traction_innkeeper:
     wallet_key: change-me
     print_key: true
     print_token: true
-  reservation.expiry_minutes: 2880
+  reservation:
+    auto_approve: true
```

Comment:
Might need to be added through to the helm value yamls and stuff? Same spots as the "expiry_minutes" in a few places?

Reply:
Good catch, I completely forgot. Added the setting to the base `values.yaml` - environment specific values can override the setting, that will otherwise be set to `false`.